Sync panic: attempted to ask for a start slot greater than the next highest epoch

Debug logs:

```
{"message":"Requesting blocks...","peer":"16Uiu2HAkvznEZLYtSFYqSSDteMvek2RVx3vhw6DRBNs2bymCVyDA","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41977,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkz2o998cx55XZ7YvbLvpGAuKK8txujKEuD32NRk9bYj3W","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41975,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmUPaZNLTzSDvtENamHyBFtbGUs8TmztWDH5Q3SuySWHAE","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41980,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmHLR1vwKkAHowAnJpz4PT9oZdHRrP6PSqKpgoUER81rNy","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41978,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm7uvhALaXSfa7aMxm3mSYNTUuEZB9HjaEGsuripWEAHyJ","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41981,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkw9W2FmRJcXdts36X3CsLYUJGd2s1NneZU3nhHqvzdQk7","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41982,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmLdReJoXaGGYKJNZkadyoVtbVfKsaXvsasdM3aTHTkQSv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41983,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkuwaepeyMqdipuFQcDZZ2twVwK4aZkqznEwboe8gGPMhC","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41984,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmC5zDJVCYTKTgKQoM1jYjDFDxY9DCSLBQfRwEXhucFZs2","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41976,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmCVsokwnprH7QJyCMh9FuSVnQwV9a4hQ5RjzmZnbNyuYv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41987,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmDC5AT2rv1iL5za7tQDk6qB5fZ8jC5GB57GATfYo7FxGo","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41989,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkxBvM5i6mMZqXDxRVEWXjDdKiz3Qzg7SdGmoFtFawMenH","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41986,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41988,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm3bugmtTRtAaq17pUzyAPpAM3PLmzgEpToeyz1V8dDt66","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41979,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm8maLMjag1TAUM52zPfmLbVMGFdwUAWgoHu1HDQLR6e17","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41985,"count":64,"step":15},"severity":"DEBUG"}
{"message":"Received 61 blocks","peer":"16Uiu2HAkuwaepeyMqdipuFQcDZZ2twVwK4aZkqznEwboe8gGPMhC","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 56 blocks","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 62 blocks","peer":"16Uiu2HAkvznEZLYtSFYqSSDteMvek2RVx3vhw6DRBNs2bymCVyDA","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 58 blocks","peer":"16Uiu2HAm3bugmtTRtAaq17pUzyAPpAM3PLmzgEpToeyz1V8dDt66","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 61 blocks","peer":"16Uiu2HAmC5zDJVCYTKTgKQoM1jYjDFDxY9DCSLBQfRwEXhucFZs2","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 57 blocks","peer":"16Uiu2HAmLdReJoXaGGYKJNZkadyoVtbVfKsaXvsasdM3aTHTkQSv","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 60 blocks","peer":"16Uiu2HAmDC5AT2rv1iL5za7tQDk6qB5fZ8jC5GB57GATfYo7FxGo","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 58 blocks","peer":"16Uiu2HAmCVsokwnprH7QJyCMh9FuSVnQwV9a4hQ5RjzmZnbNyuYv","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 54 blocks","peer":"16Uiu2HAkxBvM5i6mMZqXDxRVEWXjDdKiz3Qzg7SdGmoFtFawMenH","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 61 blocks","peer":"16Uiu2HAkw9W2FmRJcXdts36X3CsLYUJGd2s1NneZU3nhHqvzdQk7","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 60 blocks","peer":"16Uiu2HAm8maLMjag1TAUM52zPfmLbVMGFdwUAWgoHu1HDQLR6e17","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 61 blocks","peer":"16Uiu2HAkz2o998cx55XZ7YvbLvpGAuKK8txujKEuD32NRk9bYj3W","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 0 blocks","peer":"16Uiu2HAmHLR1vwKkAHowAnJpz4PT9oZdHRrP6PSqKpgoUER81rNy","prefix":"initial-sync","severity":"DEBUG"}
{"error":"failed to read chunked block: stream reset","message":"Request failed, trying to round robin with other peers","peer":"16Uiu2HAmHLR1vwKkAHowAnJpz4PT9oZdHRrP6PSqKpgoUER81rNy","prefix":"initial-sync","remaining peers":14,"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmDC5AT2rv1iL5za7tQDk6qB5fZ8jC5GB57GATfYo7FxGo","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42173,"count":4,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkz2o998cx55XZ7YvbLvpGAuKK8txujKEuD32NRk9bYj3W","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41978,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmC5zDJVCYTKTgKQoM1jYjDFDxY9DCSLBQfRwEXhucFZs2","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41993,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm3bugmtTRtAaq17pUzyAPpAM3PLmzgEpToeyz1V8dDt66","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42023,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmLdReJoXaGGYKJNZkadyoVtbVfKsaXvsasdM3aTHTkQSv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42083,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmUPaZNLTzSDvtENamHyBFtbGUs8TmztWDH5Q3SuySWHAE","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42038,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm7uvhALaXSfa7aMxm3mSYNTUuEZB9HjaEGsuripWEAHyJ","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42053,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkuwaepeyMqdipuFQcDZZ2twVwK4aZkqznEwboe8gGPMhC","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42098,"count":4,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm8maLMjag1TAUM52zPfmLbVMGFdwUAWgoHu1HDQLR6e17","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42113,"count":4,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmCVsokwnprH7QJyCMh9FuSVnQwV9a4hQ5RjzmZnbNyuYv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42143,"count":4,"step":210},"severity":"DEBUG"}
{"message":"Peer is added to exclusion list","peer":"16Uiu2HAmHLR1vwKkAHowAnJpz4PT9oZdHRrP6PSqKpgoUER81rNy","prefix":"p2p","severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkvznEZLYtSFYqSSDteMvek2RVx3vhw6DRBNs2bymCVyDA","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42008,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42158,"count":4,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkw9W2FmRJcXdts36X3CsLYUJGd2s1NneZU3nhHqvzdQk7","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42068,"count":5,"step":210},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkxBvM5i6mMZqXDxRVEWXjDdKiz3Qzg7SdGmoFtFawMenH","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42128,"count":4,"step":210},"severity":"DEBUG"}
{"message":"Received 5 blocks","peer":"16Uiu2HAkuwaepeyMqdipuFQcDZZ2twVwK4aZkqznEwboe8gGPMhC","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 59 blocks","peer":"16Uiu2HAm7uvhALaXSfa7aMxm3mSYNTUuEZB9HjaEGsuripWEAHyJ","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 0 blocks","peer":"16Uiu2HAmC5zDJVCYTKTgKQoM1jYjDFDxY9DCSLBQfRwEXhucFZs2","prefix":"initial-sync","severity":"DEBUG"}
{"error":"failed to read chunked block: invalid range or step","message":"Request failed, trying to round robin with other peers","peer":"16Uiu2HAmC5zDJVCYTKTgKQoM1jYjDFDxY9DCSLBQfRwEXhucFZs2","prefix":"initial-sync","remaining peers":13,"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmDC5AT2rv1iL5za7tQDk6qB5fZ8jC5GB57GATfYo7FxGo","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":44513,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkuwaepeyMqdipuFQcDZZ2twVwK4aZkqznEwboe8gGPMhC","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":43463,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkxBvM5i6mMZqXDxRVEWXjDdKiz3Qzg7SdGmoFtFawMenH","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":43883,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmCVsokwnprH7QJyCMh9FuSVnQwV9a4hQ5RjzmZnbNyuYv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":44093,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":44303,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm3bugmtTRtAaq17pUzyAPpAM3PLmzgEpToeyz1V8dDt66","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42413,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkz2o998cx55XZ7YvbLvpGAuKK8txujKEuD32NRk9bYj3W","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":41993,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkvznEZLYtSFYqSSDteMvek2RVx3vhw6DRBNs2bymCVyDA","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42203,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm8maLMjag1TAUM52zPfmLbVMGFdwUAWgoHu1HDQLR6e17","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":43673,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmLdReJoXaGGYKJNZkadyoVtbVfKsaXvsasdM3aTHTkQSv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":43253,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmUPaZNLTzSDvtENamHyBFtbGUs8TmztWDH5Q3SuySWHAE","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42623,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm7uvhALaXSfa7aMxm3mSYNTUuEZB9HjaEGsuripWEAHyJ","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":42833,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkw9W2FmRJcXdts36X3CsLYUJGd2s1NneZU3nhHqvzdQk7","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":43043,"count":1,"step":2730},"severity":"DEBUG"}
{"message":"Received 0 blocks","peer":"16Uiu2HAkuwaepeyMqdipuFQcDZZ2twVwK4aZkqznEwboe8gGPMhC","prefix":"initial-sync","severity":"DEBUG"}
{"error":"failed to read chunked block: invalid range or step","message":"Request failed, trying to round robin with other peers","peer":"16Uiu2HAkuwaepeyMqdipuFQcDZZ2twVwK4aZkqznEwboe8gGPMhC","prefix":"initial-sync","remaining peers":12,"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmDC5AT2rv1iL5za7tQDk6qB5fZ8jC5GB57GATfYo7FxGo","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":73493,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm8maLMjag1TAUM52zPfmLbVMGFdwUAWgoHu1HDQLR6e17","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":62573,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkz2o998cx55XZ7YvbLvpGAuKK8txujKEuD32NRk9bYj3W","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":43463,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmLdReJoXaGGYKJNZkadyoVtbVfKsaXvsasdM3aTHTkQSv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":59843,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmCVsokwnprH7QJyCMh9FuSVnQwV9a4hQ5RjzmZnbNyuYv","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":68033,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":70763,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkxBvM5i6mMZqXDxRVEWXjDdKiz3Qzg7SdGmoFtFawMenH","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":65303,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkw9W2FmRJcXdts36X3CsLYUJGd2s1NneZU3nhHqvzdQk7","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":57113,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAkvznEZLYtSFYqSSDteMvek2RVx3vhw6DRBNs2bymCVyDA","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":46193,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm7uvhALaXSfa7aMxm3mSYNTUuEZB9HjaEGsuripWEAHyJ","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":54383,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAmUPaZNLTzSDvtENamHyBFtbGUs8TmztWDH5Q3SuySWHAE","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":51653,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Requesting blocks...","peer":"16Uiu2HAm3bugmtTRtAaq17pUzyAPpAM3PLmzgEpToeyz1V8dDt66","prefix":"initial-sync","req":{"head_block_root":"cmhYYphdBizVmdZoNnnnSILqTN2NqvrARVbjXnPEdTQ=","start_slot":48923,"count":1,"step":32760},"severity":"DEBUG"}
{"message":"Received 4 blocks","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","severity":"DEBUG"}
{"message":"Received 0 blocks","peer":"16Uiu2HAmLdReJoXaGGYKJNZkadyoVtbVfKsaXvsasdM3aTHTkQSv","prefix":"initial-sync","severity":"DEBUG"}
{"error":"failed to read chunked block: invalid range or step","message":"Request failed, trying to round robin with other peers","peer":"16Uiu2HAmLdReJoXaGGYKJNZkadyoVtbVfKsaXvsasdM3aTHTkQSv","prefix":"initial-sync","remaining peers":11,"severity":"DEBUG"}
{"message":"Received 0 blocks","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","severity":"DEBUG"}
{"error":"failed to read chunked block: invalid range or step","message":"Request failed, trying to round robin with other peers","peer":"16Uiu2HAmRvrEGARjqSrvDg5ae3pC3NZLAZyoFwxuqoXzH78kBcJe","prefix":"initial-sync","remaining peers":12,"severity":"DEBUG"}
panic: attempted to ask for a start slot greater than the next highest epoch

goroutine 264 [running]:
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.(*InitialSync).Start(0xc0003f1f80)
	beacon-chain/sync/initial-sync/service.go:108 +0x97d
created by github.com/prysmaticlabs/prysm/shared.(*ServiceRegistry).StartAll
	shared/service_registry.go:44 +0x23e
```

The current slot was `52467` at the time of failure.
It was syncing with 15 peers at the time.

[beacon-archival-node.log](https://github.com/prysmaticlabs/prysm/files/3861161/out.log)

--- a/beacon-chain/sync/initial-sync/round_robin.go
+++ b/beacon-chain/sync/initial-sync/round_robin.go
@@ -79,7 +79,7 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 
 			// Short circuit start far exceeding the highest finalized epoch in some infinite loop.
 			if start > helpers.StartSlot(highestFinalizedEpoch()+1) {
-				return nil, errors.New("attempted to ask for a start slot greater than the next highest epoch")
+				return nil, errors.Errorf("attempted to ask for a start slot of %d which is greater than the next highest epoch of %d", start, helpers.StartSlot(highestFinalizedEpoch()+1))
 			}
 
 			atomic.AddInt32(&p2pRequestCount, int32(len(peers)))
@@ -157,6 +157,12 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 					}
 				}
 			}
+		}
+		startBlock := s.chain.HeadSlot() + 1
+		skippedBlocks := blockBatchSize * uint64(lastEmptyRequests*len(peers))
+		if startBlock+skippedBlocks > helpers.StartSlot(finalizedEpoch+1) {
+			log.Debug("requested block range is greater than the finalized epoch")
+			break
 		}
 
 		blocks, err := request(

--- a/beacon-chain/sync/initial-sync/round_robin.go
+++ b/beacon-chain/sync/initial-sync/round_robin.go
@@ -161,7 +161,7 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 		startBlock := s.chain.HeadSlot() + 1
 		skippedBlocks := blockBatchSize * uint64(lastEmptyRequests*len(peers))
 		if startBlock+skippedBlocks > helpers.StartSlot(finalizedEpoch+1) {
-			log.Debug("requested block range is greater than the finalized epoch")
+			log.WithField("finalizedEpoch", finalizedEpoch).Debug("Requested block range is greater than the finalized epoch")
 			break
 		}
 

--- a/tools/blocktree/main.go
+++ b/tools/blocktree/main.go
@@ -35,7 +35,7 @@ var (
 type node struct {
 	parentRoot [32]byte
 	dothNode   *dot.Node
-	score map[uint64]bool
+	score      map[uint64]bool
 }
 
 func main() {

